tests/test_sidewall.py

```diff
@@ -219,7 +221,7 @@ def test_intersection_with_inside():
         angle = np.pi / 4
         # avoid vertex-edge crossing case
         try:
```

Comment:
what is the idea behind this `try: except:`?

Reply:
The polyslab is generated from random vertices. So some of polyslabs will not pass the validation. There are two main validations: 1) no vertex-vertex crossing during dilation/erosion 2) no vertex-edge crossing during dilation/erosion. 1) can be figured out analytically, but I don't have a way to find out 2) analytically yet. So all vertices generated in this test will satisfy 1), but it's unknown if they satisfy 2). This `try: except:` is to rule out all polyslabs that fails 2).